src/easier68k/core/enum/register.py

```diff
@@ -13,11 +14,29 @@ class Register:
     D5 = 5
     D6 = 6
     D7 = 7
+    # Address registers
     A0 = 8
     A1 = 9
     A2 = 10
     A3 = 11
     A4 = 12
     A5 = 13
     A6 = 14
+    # User stack pointer
     A7 = 15
+    # Program Counter
+    PC = 16
+    ProgramCounter = 16
+    # Condition Code Register (5 bits in the lower byte)
+    CCR = 17
+    ConditionCodeRegister = 17
+
+    # constants that group together the registers
+    DATA_REGISTERS = [D0, D1, D2, D3, D4, D5, D6, D7]
+    ADDRESS_REGISTERS = [A0, A1, A2, A3, A4, A5, A6, A7]
+
+    # all of the address registers that are limited to fit within the bounds of memory 2^24
+    MEMORY_LIMITED_ADDRESS_REGISTERS = [A0, A1, A2, A3, A4, A5, A6, A7, PC]
+
+    # all of the 32 bit registers
+    FULL_SIZE_REGISTERS = [D0, D1, D2, D3, D4, D5, D6, D7, A0, A1, A2, A3, A4, A5, A6, A7, PC]
```

Comment:
You include A0-A7 and PC here and based off of it's usage it seems this should just be called "ALL_REGISTERS" or similar.

Reply:
I exclude the Condition Code Register

Reply:
It cut out the second part of my comment...
I was meaning to say that the "MEMORY_LIMITED_ADDRESS_REGISTERS" is included in the FULL_SIZE_REGISTERS which seems like a contradiction, but I can't think of a better name anyway so I think it's fine.

Reply:
Maybe the naming could be better, the "Memory Limited" registers are 32 bits, but we restrict the values to 2^24, while the full size registers are all of the 32 bit registers (including data which are not restricted)

Reply:
yeah. Let's just merge if for now.